### PR TITLE
Yoni

### PR DESCRIPTION
**Pull request by yonis9**
@hashirshoaeb Hi, I converted all components to functional components with hooks. Let me know what you think.

### CODE REVIEW

**Comment by hashirshoaeb** [comment]
./src/components/AboutMe.jsx
  Line 10:19:  'setHeading' is assigned a value but never used   no-unused-vars
  Line 11:20:  'setAboutDev' is assigned a value but never used  no-unused-vars
  Line 14:21:  'seResumeURL' is assigned a value but never used  no-unused-vars

./src/components/Project.jsx
  Line 11:19:  'setHeading' is assigned a value but never used         no-unused-vars
  Line 13:26:  'setProjectsLength' is assigned a value but never used  no-unused-vars

./src/components/MainBody.jsx
  Line 12:10:  'backgroundType' is assigned a value but never used     no-unused-vars
  Line 12:26:  'setBackgroundType' is assigned a value but never used  no-unused-vars
  Line 13:19:  'setDevInfo' is assigned a value but never used         no-unused-vars
  Line 14:19:  'setDevDesc' is assigned a value but never used         no-unused-vars

./src/components/ProjectCard.jsx
  Line 13:6:  React Hook useEffect has a missing dependency: 'handleUpdatetime'. Either include it or remove the dependency array  react-hooks/exhaustive-deps

./src/components/Footer.jsx
  Line 8:19:  'setBgStyle' is assigned a value but never used  no-unused-vars

Can you remove these warnings?

File: src/components/ProjectCard.jsx [hunk 1]
```diff
@@ -1,34 +1,26 @@
...
-    };
-  }
+  const [updated_at, setUpdatedAt] = useState("0 mints");
 
-  componentDidMount = () => {
-    this.handleUpdatetime();
-  };
+  useEffect(() => {
+    handleUpdatetime()
+  }, [])
```
Comment (hashirshoaeb) [inline src/components/ProjectCard.jsx]: remove **, [ ]**

File: src/components/ProjectCard.jsx [hunk 2]
```diff
@@ -48,51 +40,50 @@ class ProjectCard extends Component {
...
+          </p>
         </div>
       </div>
-    );
-  }
+    </div>
+  );
 }
 
 class Language extends Component {
```
Comment (hashirshoaeb) [inline src/components/ProjectCard.jsx]: Can you also change Language Component to hooks?

**Comment by hashirshoaeb** [comment]
If you are comfortable with it, you can also add yourself in contributors list in readme.md on your own. :) 
And also thank you so much for your contribution.

**Comment by yonis9** [comment]
> ./src/components/AboutMe.jsx
> Line 10:19: 'setHeading' is assigned a value but never used no-unused-vars
> Line 11:20: 'setAboutDev' is assigned a value but never used no-unused-vars
> Line 14:21: 'seResumeURL' is assigned a value but never used no-unused-vars
> 
> ./src/components/Project.jsx
> Line 11:19: 'setHeading' is assigned a value but never used no-unused-vars
> Line 13:26: 'setProjectsLength' is assigned a value but never used no-unused-vars
> 
> ./src/components/MainBody.jsx
> Line 12:10: 'backgroundType' is assigned a value but never used no-unused-vars
> Line 12:26: 'setBackgroundType' is assigned a value but never used no-unused-vars
> Line 13:19: 'setDevInfo' is assigned a value but never used no-unused-vars
> Line 14:19: 'setDevDesc' is assigned a value but never used no-unused-vars
> 
> ./src/components/ProjectCard.jsx
> Line 13:6: React Hook useEffect has a missing dependency: 'handleUpdatetime'. Either include it or remove the dependency array react-hooks/exhaustive-deps
> 
> ./src/components/Footer.jsx
> Line 8:19: 'setBgStyle' is assigned a value but never used no-unused-vars
> 
> Can you remove these warnings?

Sure I can handle this. But in my opinion, if those are never get changed, they shouldn't be at the state at all.

**Comment by hashirshoaeb** [comment]
Great. Thank you so much @yonis9 . #9 has been resolved. Merging pull request now.  If you like my repo, feel free to star it and share on your social media. :)

Best,
Hashir Shoaib

**Pull request merged**